.github/workflows/release.yml

```diff
@@ -77,9 +77,11 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
```

Comment:
[nit] : If `GITHUB_TOKEN` isn't used anymore, can we remove it? Same with the related PR too.

Reply:
It will be used by `semantic-release`  to push a new tag while doing the Github release.

Reply:
I see.